dags/openshift_nightlies/scripts/install/hypershift.sh

```diff
@@ -53,8 +53,9 @@ setup(){
     export KUBECONFIG=/home/airflow/auth/config
     if [[ $INSTALL_METHOD == "osd" ]]; then
         export OCM_CLI_VERSION=$(cat ${json_file} | jq -r .ocm_cli_version)
-        if [[ ${OCM_CLI_VERSION} == "master" ]]; then
-            git clone https://github.com/openshift-online/ocm-cli
+        if [[ ${OCM_CLI_VERSION} != "null" ]]; then
+            OCM_CLI_FORK=$(cat ${json_file} | jq -r .ocm_cli_fork)
```

Comment:
@mukrishn Where is ocm_cli_fork (and rosa_cli_fork, etc) defined? I'm not seeing it as a var in the .json files

Reply:
oooooh they are in the defaults. disregard